os/android/drmhwctwo.h

```diff
@@ -171,7 +171,7 @@ class DrmHwcTwo : public hwc2_device_t {
 
    private:
     //True after validateDisplay
-    bool checkValidateDisplay;
+    bool checkValidateDisplay = false;
```

Comment:
Sorry missed this earlier, can  u have this together with rest of private member variables below.